core/put/source/dev_ui.cpp

```diff
@@ -342,6 +342,16 @@ namespace put
 
             create_render_states();
 
+            
+            style.ScaleAllSizes(DEV_UI_SCALE);
+
+            dev_ui::util_init();
+            s_initialised = true;
+
+            return true;
+        }
+        
+        ImGuiStyle& default_pmtech_style(){
```

Comment:
Should follow convention: Allman braces where the opening curly brace goes on the next line

```c++
ImGuiStyle& default_pmtech_style()
{
```

Reply:
Ahhhh I missed that. Will fix right now.